### ISSUE TITLE
New contour levels overwrite the previous contour

### ISSUE THREAD

**Issue by theogf** [JuliaPlots/Plots.jl#2269]
When having a `contourf` and a `contour` overlapping the second contour levels overlap completely the previous contour. Even if they are previously specified :
MWE:
```julia
xrange = range(-4.0,4.0,length=100)
X1 = xrange*xrange'
X2 = 1e10*sin.(-xrange*xrange')
contourf(xrange,xrange,X1',levels=100)
contour!(xrange,xrange,X2',color=:white,levels=[-1e10,0,1e-10])
```
Plots v0.27.0
Julia 1.2.0

**Comment by yha** [comment]
The behavior I see running this code is that the 100 levels are computed in the range [-10^10,10^10] of z values from the two series. This seems reasonable to me, although I think it would probably be less surprising to choose the levels according to the z-range of the first series alone. Is that the behavior you were expecting?
You can get this by explicitly supplying the levels:
```
contourf(xrange,xrange,X1',levels=range(-16,16; length=100))
```

**Comment by theogf** [comment]
The whole idea is to be free enough to have `contourf` and `contour` are different `contour` having different range. It can be more interesting (to show distributions pdf for example), that each contour has its own range.

**Comment by yha** [comment]
Yes, it would be nice to allow different series on the same subplot have different color ranges, and this is currently not supported: see #1669, #2307.

**Comment by BioTurboNick** [comment]
You can get separate contour levels displayed by excluding the series from the colorbar:

`colorbar_entry = false`

Seems like ideally though, you'd want the levels to be calculated just from the series, and the color of the levels to be determined by the colorbar.
